guide/contribute/propose.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: guide
+title: Propose a change
+description: Contribute to the guide by proposing an improvement you've made or fixing an issue
+nav_order: 1
+parent: Contribute to guide
+permalink: /guide/contribute/propose
+image: /assets/images/guide/contribute/contribute-propose-preview.jpg
```

Comment:
Image does not exist.

Reply:
Yup. On purpose, waiting for Jose to send them over, so I'll add them in a later PR or if they're done soon, then in this one.